src/Idris/AbsSyntaxTree.hs

```diff
@@ -474,6 +482,16 @@ parserCats = [IParse]
 elabCats :: [LogCat]
 elabCats = [IElab]
 
+loggingCatsStr :: String
+loggingCatsStr = unlines
+    [ (strLogCat IParse)
+    , (strLogCat IElab)
+    , (strLogCat ICodeGen)
+    , (strLogCat IErasure)
+    , (strLogCat ICoverage)
+    , (strLogCat IIBC)
+    ]
```

Comment:
A more constructive comment  might make me more inclined to change it to map. However, non-constructive comments won't.